src/main/kotlin/model/PrisonerNetwork.kt

```diff
@@ -0,0 +1,31 @@
+package uk.gov.justice.hmpps.architecture
+
+import com.structurizr.model.Container
+import com.structurizr.model.Model
+import com.structurizr.model.SoftwareSystem
+import com.structurizr.view.ViewSet
+import uk.gov.justice.hmpps.architecture.annotations.APIDocs
+import uk.gov.justice.hmpps.architecture.annotations.Tags
+
+class PrisonerNetwork private constructor() {
+  companion object : HMPPSSoftwareSystem {
+    lateinit var system: SoftwareSystem
+    lateinit var auth: Container
+
+    override fun defineModelEntities(model: Model) {
+      system = model.addSoftwareSystem("Prisoner Network", "Provides in-cell devices and access to digital services to prisoners")
+
+      auth = system.addContainer(
+        "Azure AD",
```

Comment:
I'm wondering, is this container shared in HMPPS for other purposes, or specifically provisioned for this use case?

Reply:
Great question. This is a component of the Digital Prisoner Network, and AD was provisioned to manage prisoner device login (the in-cell devices are part of this system too).

We've just started using AD as an Oauth2 provider to authenticate prisoners on the content hub. I'm sure we'll see more use for this in future across HMPPS (and why not the wider MoJ?)

Reply:
Thanks! For the wider MoJ, there's a "DOM1" Active Directory for internal staff -- I was wondering whether this is the same?

Reply:
What do you think about naming this as "Prisoner authentication service"?

Reply:
I'd like to keep naming as close to the terminology that team is using as a starting point, but I'm not too tied to that approach. How would you feel about us updating this as it matures from a system component into a shared service? I've got some upcoming conversations with the Access team which will help me understand if/how this fits with their plans which might prompt some changes.

Reply:
It looks like naming as 'Prisoner Authentication Service'is a better fit with the rest of the component names. If we could add the extra info to distinguish it from the DOM1 AD in the description that would also help. When we add the authentication systems with the Access work we can use the same form.